packages/components/src/components/Flyout/styles/FlyoutStyles.ts

```diff
@@ -0,0 +1,39 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { sv } from '#/utilities/styleUtilities';
+
+export const flyoutContentStyles = sv({
+    base:
+        'tw-block tw-bg-base tw-shadow-mid tw-border tw-border-line tw-w-full sm:tw-w-fit sm:tw-max-w-[--flyout-max-width] tw-group tw-mt-4 sm:tw-mt-0 ' +
+        // Mobile view flyout -> dialog responsiveness
+        '[body>[data-radix-popper-content-wrapper]:has(&)]:max-sm:tw-p-2 [body>[data-radix-popper-content-wrapper]:has(&)]:max-sm:tw-w-screen [body>[data-radix-popper-content-wrapper]:has(&)]:max-sm:!tw-translate-x-0 [body>[data-radix-popper-content-wrapper]:has(&)]:max-sm:!tw-translate-y-0 ' +
+        // responsive dialog background element
+        'before:tw-fixed tw-content-[" "] before:tw-top-0 before:tw-left-0 before:tw-h-screen before:tw-w-screen before:tw-bg-black before:tw-opacity-30 before:tw-z-[-1] before:tw-pointer-events-none sm:before:tw-hidden',
+    variants: {
+        rounded: {
```

Comment:
if we added a `data-rounded` we wouldn't have to use `sv` at all

Reply:
I think this is the same as what we discussed on the previous components. I'm fully with you for using data attributes when needing to style some child or parent or something as well as when indicating a actual data change like the open state.

For purely styling things like the rounding i feel like data attribute are not ideal